app/assets/stylesheets/_buttons.scss

```diff
@@ -68,6 +67,22 @@ a .button,
 
 .orange_submit_button { padding: 5px; }
 .translation_button { @extend .play_video_button; }
+.download-button,
+.vocab-download-button {
+  float: left;
+  margin-left: none;
+}
+
+.download-button {
+  height: 33px;
```

Comment:
Can these be converted over to rem units rather than px units please? I think we agreed a sprint or two ago to start using them as a standard unit.

Reply:
@Dan I did explain in my PR above. The existing 'Learn more' btn has 33px and if I was to be use rems it would be 2.0625rem which I questioned as being too finicky. Rounding to 2.1 did not mean the buttons were the same height.

Reply:
nah thats a pretty good call on our part. Happy to leave them as px for now for clarity.